ppr-ui/src/store/actions/actions-model.ts

```diff
@@ -56,6 +56,17 @@ export const setOriginalAddCollateral: ActionIF = ({ commit }, addCollateral: Ad
   commit('mutateOriginalAddCollateral', addCollateral)
 }
 
+export const setUserAccessRole: ActionIF = ({ commit }, authRoles: Array<string>): void => {
+  const accessRole = []
+
+  if (authRoles.includes('staff')) accessRole.push('STAFF')
+  else accessRole.push('CLIENT')
+
+  if (authRoles.includes('ppr')) accessRole.push('PPR')
+  if (authRoles.includes('mhr')) accessRole.push('MHR')
+  commit('mutateUserAccessRole', accessRole.join('_'))
+}
```

Comment:
Just curious, how does this role array differs from the roles in `authRoles`. 
Do the store getters between lines between 213 - 243 do the same thing?

Reply:
`authRoles` has each role separately e.g. ["staff","ppr","mhr"]. So I wanted to create a one user access combo role e.g. 'STAFF_PPR_MHR' to use it as a _key_ to get content _values_ from resources.

Reply:
Sounds good, i like it. 
Same comment as above, with this approach we might be able to remove those previously added global getters, unless they are used elsewhere 👍